examples/Runbooks/AKS/tidb-monitor.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: pingcap.com/v1alpha1
+kind: TidbDashboard
+metadata:
+  name: basic
+spec:
+  baseImage: pingcap/tidb-dashboard
+  version: latest
+
+  ## tidb cluster to be monitored
+  ## ** now only support monitoring one tidb cluster **
```

Comment:
we have https://docs.pingcap.com/tidb-in-kubernetes/stable/monitor-a-tidb-cluster/#monitor-multiple-clusters